ext/intl/tests/listformatter/listformatter_error.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+IntlListFormatter: error messages
+--EXTENSIONS--
+intl
+--FILE--
+<?php
+
+try {
+    $formatter = new IntlListFormatter('f', IntlListFormatter::TYPE_AND, IntlListFormatter::WIDTH_WIDE);
+} catch(ValueError $exception) {
+    echo $exception->getMessage() . PHP_EOL;
+}
+
+try {
+    $formatter = new IntlListFormatter('ro_thisiswaytooooooooooooooooooooooooooooooooooooooooooooolongtobevaliditneedstobeatleast157characterstofailthevalidationinthelistformattercodeimplementation', IntlListFormatter::TYPE_AND, IntlListFormatter::WIDTH_WIDE);
+} catch(ValueError $exception) {
+    echo $exception->getMessage() .  PHP_EOL;
+}
+
+$formatter = new IntlListFormatter('ro', IntlListFormatter::TYPE_AND, IntlListFormatter::WIDTH_WIDE);
+
+try {
+    echo $formatter->format([new stdClass()]);
```

Comment:
nice but would like to see more tests, e.g. an array with a NULL somewhere ... what happens if values are references ? etc..

Reply:
Sure, I've added a few more tests.